ceph.spec.in

```diff
@@ -432,14 +432,16 @@ Group:          System/Filesystems
 Requires:       ceph-base = %{_epoch_prefix}%{version}-%{release}
 Requires:       python%{_python_buildid}-pecan
 Requires:       python%{_python_buildid}-six
+Requires:       python%{_python_buildid}-requests
```

Comment:
Please drop this line, as we try to avoid redundant `Requires:`

(`ceph-mgr` requires `ceph-base` which requires `ceph-common` which requires `python3-requests`)